tests/chainerx_tests/op_utils.py

```diff
@@ -79,6 +80,27 @@ def forward_chainerx(self, inputs):
         raise NotImplementedError(
             'Op test implementation must override `forward_chainerx`.')
 
+    def run_test_forward(self, backend_config):
```

Comment:
Will work the same even if this method is not overloaded, but included it so that behaviour one does'nt have to dig into `FunctionTestBase` to know the behaviour when `forward` is skipped.